samples/csharp_dotnetcore/25.message-reaction/DeploymentTemplates/template-with-new-rg.json

```diff
@@ -151,42 +151,44 @@
                                 }
                             }
                         },
-                      {
-                        "apiVersion": "2017-12-01",
-                        "type": "Microsoft.BotService/botServices",
-                        "name": "[parameters('botId')]",
-                        "location": "global",
-                        "kind": "bot",
-                        "sku": {
-                          "name": "[parameters('botSku')]"
-                        },
-                        "properties": {
-                          "name": "[parameters('botId')]",
-                          "displayName": "[parameters('botId')]",
-                          "endpoint": "[variables('botEndpoint')]",
-                          "msaAppId": "[parameters('appId')]",
-                          "developerAppInsightsApplicationId": null,
-                          "developerAppInsightKey": null,
-                          "publishingCredentials": null,
-                          "storageResourceId": null
-                        },
-                        "resources": [
-                          {
-                            "name": "MsTeamsChannel",
-                            "type": "channels",
+                        {
+                            "apiVersion": "2017-12-01",
+                            "type": "Microsoft.BotService/botServices",
+                            "name": "[parameters('botId')]",
                             "location": "global",
-                            "apiVersion": "2018-07-12",
```

Comment:
Did you notice that here you switched from a newer apiVersion to an older one?

Reply:
That's incorrect. The lines may look a bit jumbled in the diff, but 2017-12-01 is for the bot service and 2018-07-12 is for the Teams channel.